g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/LocaleDataUtils.java

```diff
@@ -159,6 +159,38 @@ public String getDefaultRegionCode(String inputLocale, Map<String, String> regio
 		return defaultRegionMap;
 	}
 
+	/**
+	 * Get territory of a locale
+	 *
+	 * @param localeStr
+	 * @return
+	 */
+	public String getTerritory(String localeStr){
+		Locale locale = Locale.forLanguageTag(localeStr);
+		String territory = locale.getCountry();
+		if(territory == null || territory.isEmpty())//this is for supplement default region for locales like en, zh-Hans..
+			territory = getLanguageDefaultRegionMappings().get(localeStr);
```

Comment:
How about this?
```suggestion
			territory = JSONUtils.getMapFromJson(PatternUtil.getRegionFromLib(localeStr)).get(ConstantsKeys.DEFAULT_REGION_CODE);
```

Reply:
As I confirmed with @chaoq0318 and we synced yesterday, this method is not appropriate due to it use the extracted data and is used for calling by L2 API, so not appropriate to be used in the extracting process. 

I find another method which can be used here, and change code accordingly.